packages/react-components/react-toolbar/etc/react-toolbar.api.md

```diff
@@ -4,6 +4,8 @@
 
 ```ts
 
+/// <reference types="react" />
```

Comment:
Do you know why this hs been added?

Reply:
I guess that something changes in `@fluentui/react-icons` types, but honestly - I didn't not investigated. @tomi-msft do you have any ideas?